src/components/SearchBox/SearchBox.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './SearchBox.scss';
+
+class SearchBox extends Component {
+  constructor(props) {
+    super();
+
+    this.props = props;
+    this.state = {
+      value: '',
+    };
+
+    this.onInputChange = this.onInputChange.bind(this);
+  }
+
+  onInputChange(event) {
+    const { value } = event.target;
+
+    this.setState({ value });
+  }
+
+  render() {
+    const { value } = this.state;
+
+    return (
+      <div className="searchbox">
```

Comment:
Inconsistent class naming. I think you use hyphenated classNames everywhere, why use camel case here?